src/lib/integration_test_local_engine/node_constants.ml

```diff
@@ -0,0 +1,98 @@
+let default_seed_envs =
+  [ ("DAEMON_REST_PORT", "3085")
+  ; ("DAEMON_CLIENT_PORT", "8301")
+  ; ("DAEMON_METRICS_PORT", "10001")
+  ; ("CODA_LIBP2P_PASS", "") ]
+
+let default_block_producer_envs =
+  [ ("DAEMON_REST_PORT", "3085")
+  ; ("DAEMON_CLIENT_PORT", "8301")
+  ; ("DAEMON_METRICS_PORT", "10001")
+  ; ("CODA_PRIVKEY_PASS", "naughty blue worm")
+  ; ("CODA_LIBP2P_PASS", "") ]
+
+let default_snark_coord_envs ~snark_coordinator_key ~snark_worker_fee =
+  [ ("DAEMON_REST_PORT", "3085")
+  ; ("DAEMON_CLIENT_PORT", "8301")
+  ; ("DAEMON_METRICS_PORT", "10001")
+  ; ("CODA_SNARK_KEY", snark_coordinator_key)
+  ; ("CODA_SNARK_FEE", snark_worker_fee)
+  ; ("WORK_SELECTION", "seq")
+  ; ("CODA_PRIVKEY_PASS", "naughty blue worm")
+  ; ("CODA_LIBP2P_PASS", "") ]
+
+let default_seed_command ~runtime_config =
+  [ "daemon"
```

Comment:
Ditto for the commands (these can also be written more compositionally, I think). You may even want to consider codifying the cli arguments as a typed record with options for the various flags, and writing a single function `cli_args -> string list` that actually renders this record to the untyped cli arguments.